docs/cluster-management/configure-api.md

```diff
@@ -143,23 +143,35 @@ build:
 
 ### Bookend Plugins
 
-You can globally configure which built-in bookend plugins will be used during a build. By default, `scm` is enabled to begin builds with a SCM checkout command.
+You can globally configure which built-in bookend plugins will be used during a build. Bookend plugins can be configured for each cluster.
+By default, `scm` is enabled to begin builds with a SCM checkout command.
 
 If you're looking to include a custom bookend in the API, please refer [here](#extending-the-docker-container).
 
 | Key | Default| Description |
 |:----|:-------|:------------|
-| BOOKENDS_SETUP | None | The ordered list of plugins to execute at the beginning of every build. Take the forms of `'["first", "second", ...]'` |
-| BOOKENDS_TEARDOWN | None | The ordered list of plugins to execute at the end of every build. Take the forms of `'["first", "second", ...]'` |
+| BOOKENDS | None | The associative array of bookends to be executed at the beginning and end of every build. Take the forms of `{"default": {"setup": ["scm", ...], "teardown": [...]}, "clusterA": {"setup": ["scm", ...], "teardown": [...]}}` |
 
 
 ```yaml
 # config/local.yaml
 bookends:
+  default: 
     setup:
-        - scm
-        - my-custom-bookend
+      - scm
+      - my-custom-bookend
+    teardown:
+      - screwdriver-artifact-bookend
+      - screwdriver-cache-bookend
+  clusterA:
```

Comment:
are bookends specified here merged with default or will it be a complete replacement ?

Reply:
The bookends specified here will be replaced. They are not merged with the default bookends.
The bookends defined in clusterA will be executed.